### PR TITLE
Chrome

### PR DESCRIPTION
**Pull request by APMichael**
Merged Chrome-based (7Star, 360 Browser, Coowon, SlimJet, SuperBird) entries.

Added outdated Chrome-based (7Star, 360 Browser, Coowon, SlimJet, SuperBird) entries to "Removed entries.ini": #194

I didn’t touch the entry [Comodo Dragon *]:
- Some of the FileKeys differ too much.
- In older versions of winapp2.ini the FileKeys were merged already, but someone moved them out into one big single entry again.

### CODE REVIEW

File: Winapp2.ini [hunk 1]
```diff
@@ -441,105 +325,131 @@ FileKey5=%ProgramFiles%\Google\Chrome\Application\*\Installer|*.7z
 [JumpListIconsRecentClosed *]
```
Comment (MoscaDotTo) [inline Winapp2.ini]: I think this entry should perhaps be merged into History *

Comment (MoscaDotTo) [reply Winapp2.ini]: Addtly, it looks like there's another path here called "JumpListIconsMostVisited" that can be similarly cleaned. Both are for the jumplist that opens when you right click chrome on the windows task bar.

File: Winapp2.ini [hunk 2]
```diff
@@ -441,105 +325,131 @@ FileKey5=%ProgramFiles%\Google\Chrome\Application\*\Installer|*.7z
...
 LangSecRef=3029
 SpecialDetect=DET_CHROME
+Detect1=HKCU\Software\7Star
+Detect2=HKCU\Software\Coowon
+Detect3=HKCU\Software\SuperBird
 Default=False
-FileKey1=%LocalAppData%\Google\Chrome\User Data\Default\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey1=%LocalAppData%\7Star\7Star\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey2=%LocalAppData%\Coowon\Coowon\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey3=%LocalAppData%\Google\Chrome\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
```
Comment (MoscaDotTo) [inline Winapp2.ini]: \Chrome\ here should be \Chrome*\ for Chrome SxS

**Comment by APMichael** [comment]
Thanks for your replies. I'm not an expert for Chrome. Therefore, I just merged and moved the separated Chrome-based entries, using the remaining entries as reference. I updated the pull request with your suggestion.

**Comment by MoscaDotTo** [comment]
LGTM 👍

**Pull request merged**